mhs-reference-implementation/selenium_tests/page_objects/build_scr.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from src.gp_summary_update import SummaryCareRecord
+from utilities.message_utilities import MessageUtilities
```

Comment:
As we're following the google standards I think we're trying to avoid importing specific classes in favor of importing the whole module.